catroid/src/org/catrobat/catroid/formulaeditor/Sensors.java

```diff
@@ -32,7 +32,7 @@
 	PHIRO_SIDE_RIGHT, PHIRO_BOTTOM_LEFT, PHIRO_BOTTOM_RIGHT, DRONE_BATTERY_STATUS, DRONE_EMERGENCY_STATE, DRONE_FLYING,
 	DRONE_INITIALIZED, DRONE_USB_ACTIVE, DRONE_USB_REMAINING_TIME, DRONE_CAMERA_READY, DRONE_RECORD_READY,
 	DRONE_RECORDING, DRONE_NUM_FRAMES, OBJECT_X_VELOCITY(true), OBJECT_Y_VELOCITY(true), OBJECT_ANGULAR_VELOCITY(true),
-	LAST_FINGER_INDEX, FINGER_X, FINGER_Y, FINGER_TOUCHED;
```

Comment:
you removed the constants for the multi touch sensors, hence it does not compile. please have a look at the Jenkins results for details. however, in short: you should add these constants again as they are needed.